package.json

```diff
@@ -7,6 +7,7 @@
     "npm": ">=3.8"
   },
   "dependencies": {
+    "babel-plugin-transform-builtin-extend": "^1.1.2",
```

Comment:
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error#ES6%20Custom%20Error%20Class and https://github.com/loganfsmyth/babel-plugin-transform-builtin-extend, which is needed so that babel can handle the custom error classes